modules/solid_properties/test/tests/graphite/test.i

```diff
@@ -0,0 +1,83 @@
+[Mesh]
+  type = GeneratedMesh
```

Comment:
I don't know what the current convention is - will `GeneratedMesh` eventually be deprecated? There's loads of tests still using `GeneratedMesh`. Happy to update this though!

Reply:
I know at one point there was a big push to migrate all tests that used mesh modifiers (which were deprecated) to use the mesh generator syntax (mesh modifiers were replaced by equivalent mesh generators). My understanding was to migrate to mesh generators for all tests. I know there are still many in the repository using `GeneratedMesh`. I don't know what MOOSE's convention is, and someone on the MOOSE team should chime in here, but I know any new inputs I create I use the `GeneratedMeshGenerator` syntax rather than allow new inputs with the old syntax if it is supposed to be updated in the future.

Reply:
Thanks for explaining - I update all the tests to use mesh generators.